tests/integration/upgrade-integration-test.sh

```diff
@@ -28,17 +29,15 @@ else
 fi
 
 echo 'Testing signed official docker image...'
-kubectl run dpod --image=docker.io/library/hello-world -lapp.kubernetes.io/instance=connaisseur >output.log 2>&1 || true
+kubectl run dpod-${RAND} --image=docker.io/library/hello-world -lapp.kubernetes.io/instance=connaisseur >output.log 2>&1 || true
 
-if [[ "$(cat output.log)" != 'pod/dpod created' ]]; then
+if [[ "$(cat output.log)" != 'pod/dpod-${RAND} created' ]]; then
   echo 'Failed to allow signed image. Output:'
   cat output.log
   exit 1
 else
   echo 'Successfully allowed usage of signed image'
 fi
 
-kubectl delete pods --all || { echo 'Failed to delete test pods'; exit 1; }
```

Comment:
Can you put this behind a GITHUB_ACTIONS flag like here: https://github.com/sse-secure-systems/connaisseur/blob/develop/tests/integration/load-test.sh#L45

Then you can still run it locally without polluting your env, but still gain speed in the CI?

Reply:
done